src/server/config.rs

```diff
@@ -141,6 +146,9 @@ impl Default for Config {
             heavy_load_threshold: 300,
             // The resolution of timer in tokio is 1ms.
             heavy_load_wait_duration: ReadableDuration::millis(1),
+            enable_request_batch: true,
+            request_batch_enable_cross_command: true,
+            request_batch_wait_duration: ReadableDuration::millis(1),
```

Comment:
I think we should not wait by default, only need wait when we know the underlying layer is under heavy workload.